src/components/icon/icons/bulb.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Bulb({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest} viewBox={'0 0 12 20'}>
+      <g stroke="none" strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
```

Comment:
Can probably remove one row of `g` by combining the two to something like:

`<g stroke="none" strokeWidth={strokeWidth} fillRule="nonzero" fill={fill}>`